src/Hydrator/EloquentHydrator.php

```diff
@@ -115,6 +115,32 @@ public function __construct(HttpServiceInterface $service)
     {
         $this->service = $service;
     }
+    
+    /**
+     * @param StandardObjectInterface $attributes
+     * @param $record
+     * @return void
+     */
+    protected function defaultHydrateAttributes(StandardObjectInterface $attributes, $record)
```

Comment:
this needs to be called `attributeKeys` for consistency with a trait that exists (which will at some point be used in this `EloquentHydrator`). The method signature is correct.